app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self.__favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self) -> str:
```

Comment:
return type is None

Reply:
If I change to None here, then I have warning in main.py - Expected type 'list[str]', got 'list[None]' instead
I am puzzled how to fix it